### ISSUE TITLE
A question about the epv_pretrain_ckpt.

### ISSUE THREAD

**Issue by xiehousen** [brain-research/unsupervised-hierarch-video-prediction#3]
hello , I am very interested in your work, I also want to restore the effect of modern code, so I downloaded the Rebot dataset, but when I run the code, I found that there is no epv_pretrain_ckpt, and I still can't access the URL. Where can I find this? The download address, or can you send me, thank you very much!
My mailbox is xiehousen@gmail.com
![default](https://user-images.githubusercontent.com/37787619/50445579-a14e3d80-094a-11e9-97ab-4ebc469eef42.png)

**Comment by wichersn** [comment]
Make sure you followed the instructions in [the Use data on Google Cloud of the readme](https://github.com/brain-research/long-term-video-prediction-without-supervision#use-data-on-google-cloud).

Also, what is the output when you run
`gsutil cp -r gs://unsupervised-hierarch-video/pretrained_models/epva_human ~/Downloads
`
?

**Comment by xiehousen** [comment]
> Make sure you followed the instructions in [the Use data on Google Cloud of the readme](https://github.com/brain-research/long-term-video-prediction-without-supervision#use-data-on-google-cloud).
> 
> Also, what is the output when you run
> `gsutil cp -r gs://unsupervised-hierarch-video/pretrained_models/epva_human ~/Downloads `
> ?

I use the Robot push dataset ,I download the dataset from the URL that you give.And add the --data_dir in the terminal. 
![default](https://user-images.githubusercontent.com/37787619/50469605-2176b080-09e8-11e9-8147-1bdbf1fae7f5.png)
I see that your code uses the pre-training model by default. Can you use this pre-training model only by using open the generation.ipynb file. That‘s can be found in Inference on pretrained models.
Thank you for your reply.

**Comment by wichersn** [comment]
You're right, the the pretrain checkpoint shouldn't be used with the robot push dataset, that was a problem with the instructions. I updated the instructions for the robot push dataset to have `--epv_pretrain_ckpt ''` which will run without loading the checkpoint.

**Comment by funkydvd** [comment]
Does the bucket still exist? I receive BucketNotFoundException: 404 gs://unsupervised-hierarch-video bucket does not exist. Any chance to re-upload it or to send me via e-mail? I would be very thankful. It is for an university project.

**Comment by kathrinawu** [comment]
> Does the bucket still exist? I receive BucketNotFoundException: 404 gs://unsupervised-hierarch-video bucket does not exist. Any chance to re-upload it or to send me via e-mail? I would be very thankful. It is for an university project.

Same problem for me. Can not access the pretrained models
